packaging/nntrainer.spec

```diff
@@ -69,6 +69,8 @@ BuildRequires:	python3
 BuildRequires:	python3-numpy
 
 BuildRequires:	%{capi_machine_learning_common}-devel
+BuildRequires:	%{capi_machine_learning_inference}-devel
+Requires:	%{capi_machine_learning_inference}
```

Comment:
1. Do we have "capi-nnstreamer-devel" or "capi-nnstreamer" packages in Tizen? (it is capi-machine-learning-inference.rpm)
2. No need to declare "Requires" for shared libraries.

Reply:
1. For tizen 6.0, we have `capi-nnstreamer(-devel)`. For tizen >= 6.5 `capi-machine-learning-inference` 
2. Thank you for the notice :) Requires section deleted.